client/admin/components/users/ImportDialog.vue

```diff
@@ -137,39 +168,46 @@ export default {
     showDialog(val) {
       this.$nextTick(() => this.focusTrap.toggle(val));
     }
+  },
+  mounted() {
+    window.addEventListener('dragenter', this.showDropZone);
   }
 };
 </script>
 
 <style lang="scss" scoped>
```

Comment:
Run linter to check if everything is in order.

Reply:
Linting is active, but order defined in [.eslintrc.js](https://github.com/ExtensionEngine/boutique/blob/develop/.eslintrc.js) differs from the default one and one defined in Vue style guide.

Reply:
@vladimyr ? Do we have inconsistent linting across projects?

Reply:
@vladimyr ? Do we have inconsistent linting across projects?